test.rb

```diff
@@ -1,2 +1,13 @@
 puts 'Hello World'
 puts "hollow world"
+puts "More Badly formatted" +  "Code to flag"  + "rubocop Warnings"
```

Comment:
C: Prefer single-quoted strings when you don't need string interpolation or special symbols.